### ISSUE TITLE
Doesn't build on 8.11

### ISSUE THREAD

**Issue by gmalecha** [unicoq/unicoq#40]
I can't seem to get this repository to build against 8.11.0. I checked out:c33e66c8f2924449c7b98aab108d97b5ee105bab per @Janno 's suggestion but i still get the following.

```
$ make -j3
CAMLDEP src/munify.ml
coq_makefile -f Make -o Makefile
CAMLOPT -c -for-pack Unicoq src/munify.ml
CAMLOPT -c -for-pack Unicoq src/unitactics.ml
CAMLOPT -pack -o src/unicoq.cmx
CAMLOPT -a -o src/unicoq.cmxa
CAMLOPT -shared -o src/unicoq.cmxs
COQC theories/Unicoq.v
File "./theories/Unicoq.v", line 7, characters 0-27:
Error: while loading
/home/gmalecha/bedrock/mtac-testing/unicoq/src/unicoq.cmxs:
The module `Int_misc' is already loaded (either by the main program or a previously-dynlinked library)

make[1]: *** [Makefile:678: theories/Unicoq.vo] Error 1
make: *** [Makefile:327: all] Error 2
```

**Comment by Janno** [comment]
As you'll hopefully see on Zulip I've made an off-by-one error and gave you the commit just before the one I should have given you: d1ff6f05a1740543623374b94a07ca1abfbd4ea9.

*However*, while that commit will fix universe issues, it will not fix what you have encountered here. I've never seen this error, I think. I would advise cleaning up the directory and building without `-j`, just in case. If the error persists you'll have to ask the Coq developers. Somebody might have seen similar errors.

I should note that I mostly use OCaml 4.07.1. Which version are you using? 4.08 and up will not work with Mtac2, as far as I know (for reasons that haven't had time to investigate yet). Not sure if the same applies to Unicoq.

**Comment by beta-ziliani** [comment]
There is a tag for 8.11 (v1.3-8.11). However, as Janno pointed out, that won't help you here. There is an long standing issue with certain versions of OCaml that I was never able to grok. In any case, we should fix it, so let me know the details of your setup please.

**Comment by beta-ziliani** [comment]
@gmalecha did you still have this issue? I just run into this issue here, but after a `make clean` and `make` it worked :shrug:

**Comment by beta-ziliani** [comment]
Closing it until we have more evidence that the problem is still occurring.

**Comment by Janno** [comment]
I think (part of?) the problem should be fixed by #47 but I don't remember exactly :(
